src/SOS/SOS.Extensions/MemoryServiceFromDebuggerServices.cs

```diff
@@ -31,6 +31,7 @@ internal MemoryServiceFromDebuggerServices(ITarget target, DebuggerServices debu
             {
                 case Architecture.X64:
                 case Architecture.Arm64:
+                case (Architecture)6 /* Architecture.LoongArch64 */:
```

Comment:
From .NET7.0, `Architecture.LoongArch64` is ok.

Reply:
ok，I'll synchronize it

Reply:
> From .NET7.0, `Architecture.LoongArch64` is ok.

I have already synchronized.

Reply:
But there are some building erros as the CI's SDK is lower than SDK7.0.
Maybe we have to revert it.